src/theory/quantifiers/ceg_t_instantiator.cpp

```diff
@@ -1326,8 +1332,16 @@ Node BvInstantiator::rewriteAssertionForSolvePv(CegInstantiator* ci,
           ret = cur;
         }
       }
-      // careful that rewrites above do not affect whether this term contains pv
-      visited_contains_pv[cur] = contains_pv;
+
```

Comment:
Maybe write a comment here to explain why the recomputation of contains pv is necessary? You had an example right?

Reply:
Done.